net-mgmt/snmp/pkg-descr

```diff
@@ -0,0 +1,6 @@
+ bsnmp. It allows you to use SNMPv1 and SNMPv2 to query your OPNsense appliance
```

Comment:
Allow to use .... ~~your OPNsense appliance~~...

Possibly scrape the first bits of the manual page of the service or reuse the FreeBSD package descriptions altogether.